### PR TITLE
Allow displaying only enabled rules in RulesCommand

### PR DESCRIPTION
**Pull request by natanrolnik**
(Previously opened as #1275)
See issue #1270.
I've also updated `Commandant` to include the fix for `Switch`es [included in this PR](https://github.com/Carthage/Commandant/pull/96).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -33,6 +33,10 @@
   [#598](https://github.com/realm/SwiftLint/issues/598)
   [#975](https://github.com/realm/SwiftLint/issues/975)
 
+* Allow displaying only enabled rules in `RulesCommand`.
```
Comment (jpsim) [inline CHANGELOG.md]: Could you please add the CLI flag to pass in here?

File: Source/swiftlint/Commands/RulesCommand.swift [hunk 2]
```diff
@@ -42,27 +42,53 @@ struct RulesCommand: CommandProtocol {
...
 
-    // swiftlint:disable:next line_length
     static func evaluate(_ mode: CommandMode) -> Result<RulesOptions, CommandantError<CommandantError<()>>> {
         return create
             <*> mode <| configOption
             <*> mode <| Argument(defaultValue: "",
                                  usage: "the rule identifier to display description for")
+            <*> mode <| Switch(flag: "e",
+                               key: "enabled",
+                               usage: "only enabled rules are displayed")
```
Comment (jpsim) [inline Source/swiftlint/Commands/RulesCommand.swift]: "only display enabled rules"?

**Comment by NatanRolnik** [comment]
@jpsim rebased and changed according to your suggestions.

**Comment by marcelofabri** [comment]
🚢

**Pull request merged**